softdevice_controller/CHANGELOG.rst

```diff
@@ -15,9 +15,12 @@ Master branch
 Added
 =====
 
+* Added support for multiple advertising sets (DRGN-15426).
 * Added radio front-end module (FEM) support, based on the :ref:`mpsl_fem` (DRGN-11059).
 * Added support for the vendor specific HCI command: Read Supported Vendor Specific Commands (DRGN-13763).
 * Added support for the vendor specific HCI command: Zephyr Read Key Hierarchy Roots (DRGN-13237).
+* On nRF53 Series devices, the maximum supported radio output power is increased from 0 dBm to 3 dBm.
+  In this case, NRF_VREQCTRL->VREGRADIO.VREQH will be set (DRGN-15476).
```

Comment:
What do we mean with "in this case"? If the maximum supported radio output power is increased?

Reply:
If the output power is above 0 dBm, NRF_VREQCTRL->VREGRADIO.VREQH will be set

Reply:
OK, then let's say that. :)
```suggestion
  If the output power is above 0 dBM, NRF_VREQCTRL->VREGRADIO.VREQH is set (DRGN-15476).
```